index.html

```diff
@@ -45,8 +45,48 @@
 	</head>
 
 	<body>
-		<article>
-			<ts-button>Button</ts-button>
+		<article style="height: 300px">
+			<ts-button id="button">Button</ts-button>
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<br />
+			<ts-select id="select" multiselect placeholder="Choose your destiny"></ts-select>
 		</article>
+		<script>
+			(function () {
+				const select = document.getElementById('select');
+				select.items = [
```

Comment:
I'm not quite sure if we should prefer properties over slots for cases like this or not. This is still something that we have to discuss and find a clear answer for. If we ever find time for that :D 
Maybe in future we can improve this component to let them add options with slots as well.

Reply:
Yeah, I also think, that we could provide just a slot and allow developers put their select items. But I think it would be tricky with multi-level component, plus I'm not sure what benefits it will give developers if we will provide slots instead of API, because at the end they will have the same data model (array of objects with names and ids).
Anyway, we should discuss it first :)